lib/tldr-lint.js

```diff
@@ -59,7 +60,7 @@ module.exports.ERRORS = parser.ERRORS = {
     parser.yy.page.description.push(description);
   };
   parser.yy.addMoreInformation = function(url) {
-    parser.yy.page.moreInformation = url;
+    parser.yy.page.moreInformation.push(url);
```

Comment:
I would actually call this variable "infoLink" or something like that. It's more specific than "More information" without being too specific like "Homepage", since the link may be to a manpage, an announcement blog post, etc.